### PR TITLE
rev06 updates

### PR DESCRIPTION
**Pull request by jrs65**
- config(daily): fix thermal gain correction application
- config(daily): mask an extra RFI band to improve delay spectra
- config(daily): add an extra test day
- config(daily): change distribution to reduce allocation usage

### CODE REVIEW

**Comment by jrs65** [comment]
@sjforeman can you push your suggested ThesholdVisWeight config changes into this branch?

**Review by ljgray — APPROVED** [review]

File: ch_pipeline/processing/daily.py [hunk 1]
```diff
@@ -141,10 +141,23 @@
...
+      requires: manager
       in: tstream
+      out: full_bad_baseline_mask
+      params:
+        average_type: "median"
+        absolute_threshold: 1e-7
+        relative_threshold: 1e-5
+        pols_to_flag: "copol"
+        save: true
+        output_name: "bad_baseline_mask_{{tag}}.h5"
```
Comment (jrs65) [inline ch_pipeline/processing/daily.py:155]: I think we elected not to save these out, right?

Comment (sjforeman) [reply ch_pipeline/processing/daily.py:155]: Oops sorry, I completely misread your Slack message! force-push incoming...

Comment (jrs65) [reply ch_pipeline/processing/daily.py:155]: 🚅

**Review by sjforeman — DISMISSED** [review]

**Comment by ljgray** [comment]
Masking out the bright point sources would close chime-experiment/Pipeline#65 I believe

**Pull request merged**